gatsby-node.js

```diff
@@ -57,8 +59,9 @@ const createArticles = async (createPage) => {
   })
 }
 
-const createAutors = async (createPage, slug) => {
+const createAuthors = async (createPage, slug) => {
   const resp = await axios.get(`${BASE_URL}https://www.thedp.com/staff/${slug}.json`)
```

Comment:
actually, can we write a utils function to abstract out url => axios.get(`${BASE_URL}${url}`